VirtualH89/Src/GeneralPurposePort.h

```diff
@@ -116,131 +116,6 @@ class GeneralPurposePort: public virtual IODevice
     /// Enable Single Step Interrupt - not sure about this.
     static const BYTE gpp_SingleStepInterrupt_c = 0x01;
 
-    /// Enable the Timer Interrupt
-    static const BYTE gpp_EnableTimer_c         = 0x02;
-
-    /// Disable the ROM and map the RAM to the lower 8k.
-    static const BYTE gpp_DisableROM_c          = 0x20;
-
-    /// Side-select for the floppy drive (appears to be just for Hard-sectored controllers).
-    static const BYTE gpp_SideSelect_c          = 0x40;
-
-    /// Third party add-on for speed control
-    static const BYTE gpp_4MHz_2MHz_Select_c    = 0x10;
-
-    /// Magnolia Microsystems 128K RAM board.
-    static const BYTE gpp_Mms_128k_Bit2_c       = 0x04;
-    static const BYTE gpp_Mms_128k_Bit4_c       = 0x10; // note: conflict with 2/4 MHz
-    static const BYTE gpp_Mms_128k_Bit5_c       = 0x20;
-
-    ///
```

Comment:
I was referring to the comments below. These apply directly to the MMS77318, and give a nice overview on how it functions. This info was pull from the MMS 128K distribution disk.